decidim-core/db/migrate/20240326104215_remove_show_in_footer_in_static_pages.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveShowInFooterInStaticPages < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :decidim_static_pages, :show_in_footer, :Boolean
```

Comment:
The field type should be downcase

```suggestion
    remove_column :decidim_static_pages, :show_in_footer, :boolean
```